services/movie_session.py

```diff
@@ -0,0 +1,45 @@
+from django.db.models import QuerySet
+from datetime import datetime
+from db.models import MovieSession
+
+
+def create_movie_session(
+    movie_show_time: str, movie_id: int, cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id,
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet :
+
+    if session_date:
+        session_date = datetime.strptime(session_date, "%Y-%m-%d").date()
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=session_id)
```

Comment:
Using `get()` without handling exceptions can lead to `DoesNotExist` errors if the session ID is not found. Consider wrapping this in a try-except block to handle such cases gracefully.